src/providers/apolloClient.tsx

```diff
@@ -125,25 +124,27 @@ const errorLink = onError(({ operation, graphQLErrors, networkError }) => {
         user: sentryUser(),
       })
     );
-  }
-
-  if (networkError) {
+  } else if (networkError) {
```

Comment:
We were double-sending before because errors woudl have both the graphql error and the network error. Change to else if to avoid that